tests/html_attributes/style.html

```diff
@@ -0,0 +1,30 @@
+<div style="
+color:
+#fFf
+"></div>
+
+<div style=" "></div>
+<div style></div>
+
+<div style="
+all: initial;display: block;
+contain: content;text-align: center;
+background: linear-gradient(to left, hotpink, #FFF00F, #ccc, hsla(240, 100%, 50%, .05), transparent);
+max-width: 500px;margin: 0 auto;
+border-radius: 8px;transition: transform .2s ease-out;
+"></div>
+
+
+
+<div style="
+background: linear-gradient(to left, hotpink, #FFF00F, #ccc, hsla(240, 100%, 50%, .05), transparent);
+"></div>
+
+<div style="   color : red;
+            display    :inline ">
+  </div>
+
+<div attribute-1 attribute-2 attribute-3 attribute-4 attribute-5 attribute-6 attribute-7 
+style="css-prop-1: css-value;css-prop-2: css-value;css-prop-3: css-value;css-prop-4: css-value;"
+ attribute-1 attribute-2 attribute-3 attribute-4 attribute-5 attribute-6 attribute-7 >
+  </div>
```

Comment:
Let's add test for `<div style="{{ ...styles }}" ></div>`

Reply:
This is not good one... since `split(/\s/).join(' ')` the same....

Reply:
Can we the add test for that case?

Reply:
What language of this syntax (I need put in right place)? Can you give me another fixture?

Reply:
Is this `<div style="{{ ...styles['a    b'] }}" ></div>` valid code? If so, spaces between `a` and `b` should matters.

Reply:
Other example: `<div style="{{ getStylesFrom(styles) }}" ></div>`, so better ignore style tag starts with `{{` and `}}`

Reply:
> This is not a good one... since `split(/\s/).join(' ')` the same.... haha

Sorry I said this, I thought we are on another PR https://github.com/prettier/prettier/pull/7555 .

Reply:
No, as I said above developers can use `{{ ...styles }}` in `style` attribute so we should add tests for which cases to ensure we don't break code

Reply:
Tests added https://github.com/prettier/prettier/pull/7556/commits/8d78ef78e2bb8b048ba88989c515fb2748969389 .

I don't know this syntax well , can `{{` at any position or just start ?

Reply:
On old vue versions it can be in any places

Reply:
OK, updating logic